docs/UserGuide.md

```diff
@@ -221,6 +221,19 @@ Format: `link ev/EVENT_NAME c/CONTACT_NAME [c/MORE_CONTACT_NAMES]`
  c/John Doe c/Bob Dylan` will link `Alice Black` to the event `NUS Career Fest` while
   `John Doe` and `Bob Dylan` will not be linked.
 
+### Selecting an event: `select_event`
+
+Selects a specified event from JobFestGo. The relevant contacts and tasks will be displayed.
+
+Format: `select_event INDEX`
+
+* Selects the event at the specified `INDEX`. 
+* The index refers to the index number shown in the displayed events list.
+* The index **must be a positive integer** 1, 2, 3, …​
```

Comment:
is listing 1, 2, 3 necessary? and should we be ending with a full stop rather than ellipsis?

Reply:
I was following the format that was used for deleting and editing with index, but if it's agreed upon, then we can change everything.

Reply:
I understand ok then I shall ignore we shall stick to the format for now